app/main.py

```diff
@@ -1 +1,45 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is a class variable that tracks all living animals. Ensure that it is correctly updated when an animal's health changes.